src/services/telemetry/TelemetryService.test.ts

```diff
@@ -27,7 +27,6 @@ describe("Telemetry system is abstracted and can easily switch between providers
 	})
 	const MOCK_USER_INFO = {
 		id: "test-user-123",
-		email: "test@example.com",
```

Comment:
User auth object usually included user's email and I believe we actually use it to identify users.